<?xml  version="1.0"  encoding="UTF-8"?>
<ui  version="4.0">
  <class>Form</class>
  <widget  class="QWidget"  name="Form">
    <property  name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>906</width>
        <height>549</height>
      </rect>
    </property>
    <property  name="windowTitle">
      <string>Form</string>
    </property>
    <layout  class="QHBoxLayout"  name="horizontalLayout">
      <item>
        <widget  class="QSplitter"  name="splitter">
          <property  name="orientation">
            <enum>Qt::Horizontal</enum>
          </property>
          <widget  class="QWidget"  name="wA">
            <property  name="sizePolicy">
              <sizepolicy  hsizetype="Preferred"  vsizetype="Preferred">
                <horstretch>8</horstretch>
                <verstretch>0</verstretch>
              </sizepolicy>
            </property>
            <layout  class="QVBoxLayout"  name="vlA">
<!--              <item>
                <widget  class="QMdiArea"  name="mdiArea"/>
              </item>-->
            </layout>
          </widget>
          <widget  class="QWidget"  name="wB">
            <property  name="sizePolicy">
              <sizepolicy  hsizetype="Minimum"  vsizetype="Minimum">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
              </sizepolicy>
            </property>
            <layout  class="QVBoxLayout"  name="vlB">
              <item>
                <widget  class="QFrame"  name="wBA">
                  <property  name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property  name="frameShadow">
                    <enum>QFrame::Raised</enum>
                  </property>
                  <property  name="lineWidth">
                    <number>8</number>
                  </property>
                  <layout  class="QVBoxLayout"  name="vlBA">
                    <item>
                      <widget  class="QLabel"  name="label_select">
                        <property  name="text">
                          <string>Selected  Columns</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <layout  class="QHBoxLayout"  name="horizontalLayout_5">
                        <item>
                          <spacer  name="horizontalSpacer_5">
                            <property  name="orientation">
                              <enum>Qt::Horizontal</enum>
                            </property>
                            <property  name="sizeHint"  stdset="0">
                              <size>
                                <width>40</width>
                                <height>20</height>
                              </size>
                            </property>
                          </spacer>
                        </item>
                        <item>
                          <layout  class="QVBoxLayout"  name="vlBAA"/>
                        </item>
                      </layout>
                    </item>
                    <item>
                      <layout  class="QHBoxLayout"  name="vlBAB">
                        <item>
                          <spacer  name="horizontalSpacer">
                            <property  name="orientation">
                              <enum>Qt::Horizontal</enum>
                            </property>
                            <property  name="sizeHint"  stdset="0">
                              <size>
                                <width>40</width>
                                <height>20</height>
                              </size>
                            </property>
                          </spacer>
                        </item>
                        <item>
                          <widget  class="QCheckBox"  name="check_all">
                            <property  name="text">
                              <string>All</string>
                            </property>
                          </widget>
                        </item>
                        <item>
                          <widget  class="QCheckBox"  name="check_inv">
                            <property  name="text">
                              <string>Invert</string>
                            </property>
                          </widget>
                        </item>
                        <item>
                          <widget  class="QPushButton"  name="selBtn">
                            <property  name="text">
                              <string>Select</string>
                            </property>
                          </widget>
                        </item>
                      </layout>
                    </item>
                  </layout>
                </widget>
              </item>
              <item>
                <widget  class="QFrame"  name="wBB">
                  <property  name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property  name="frameShadow">
                    <enum>QFrame::Raised</enum>
                  </property>
                  <property  name="lineWidth">
                    <number>3</number>
                  </property>
                  <layout  class="QVBoxLayout"  name="vlBB">
                    <item>
                      <widget  class="QLabel"  name="label_sort">
                        <property  name="text">
                          <string>Sort  By</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <layout  class="QVBoxLayout"  name="vlBBA">
                      <item>
                        <layout  class="QHBoxLayout"  name="vlTmp">
                            <item>
                            <spacer  name="horizontalSpacer_2">
                                <property  name="orientation">
                                <enum>Qt::Horizontal</enum>
                                </property>
                                <property  name="sizeHint"  stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                                </property>
                            </spacer>
                            </item>
                            <item>
                            <widget  class="QCheckBox"  name="check_sort1">
                                <property  name="text">
                                <string>Sort  col1</string>
                                </property>
                            </widget>
                            </item>
                            <item>
                            <widget  class="QCheckBox"  name="check_asc1">
                                <property  name="text">
                                <string>Ascending</string>
                                </property>
                            </widget>
                            </item>
                        </layout>
                        </item>
                        <item>
                          <layout  class="QVBoxLayout"  name="vlBBAA"/>
                        </item>
                      </layout>
                    </item>
                    <item>
                      <layout  class="QVBoxLayout"  name="vlBBB">
                        <item>
                          <layout  class="QHBoxLayout"  name="vlTmp2">
                                <item>
                                <spacer  name="horizontalSpacer_3">
                                    <property  name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property  name="sizeHint"  stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                    </property>
                                </spacer>
                                </item>
                                <item>
                                <widget  class="QCheckBox"  name="check_sort2">
                                    <property  name="text">
                                    <string>Sort  col2</string>
                                    </property>
                                </widget>
                                </item>
                                <item>
                                <widget  class="QCheckBox"  name="check_asc2">
                                    <property  name="text">
                                    <string>Ascending</string>
                                    </property>
                                </widget>
                                </item>
                          </layout>
                        </item>  
                        <item>
                          <layout  class="QVBoxLayout"  name="vlBBBA"/>
                        </item>
                      </layout>
                    </item>
                    <item>
                      <layout  class="QHBoxLayout"  name="horizontalLayout_2">
                        <item>
                          <spacer  name="horizontalSpacer_4">
                            <property  name="orientation">
                              <enum>Qt::Horizontal</enum>
                            </property>
                            <property  name="sizeHint"  stdset="0">
                              <size>
                                <width>40</width>
                                <height>20</height>
                              </size>
                            </property>
                          </spacer>
                        </item>
                        <item>
                          <widget  class="QPushButton"  name="sortBtn">
                            <property  name="text">
                              <string>Sort</string>
                            </property>
                          </widget>
                        </item>
                      </layout>
                    </item>
                    <item>
                      <spacer  name="verticalSpacer">
                        <property  name="orientation">
                          <enum>Qt::Vertical</enum>
                        </property>
                        <property  name="sizeHint"  stdset="0">
                          <size>
                            <width>20</width>
                            <height>40</height>
                          </size>
                        </property>
                      </spacer>
                    </item>
                  </layout>
                </widget>
              </item>
            </layout>
          </widget>
        </widget>
      </item>
    </layout>
  </widget>
  <resources/>
  <connections/>
</ui>
